sklearn/semi_supervised/tests/test_label_propagation.py

```diff
@@ -135,9 +137,9 @@ def test_convergence_speed():
     assert_array_equal(mdl.predict(X), [0, 1, 1])
 
 
-def test_convergence_warning():
+def test_convergence_warning(global_dtype):
     # This is a non-regression test for #5774
-    X = np.array([[1.0, 0.0], [0.0, 1.0], [1.0, 2.5]])
+    X = np.array([[1.0, 0.0], [0.0, 1.0], [1.0, 2.5]], dtype=global_dtype)
```

Comment:
I don't think this test should be parametrized. It's just checking convergence warnings